apps/web/app/api/user/stats/emails/all/get-all.ts

```diff
@@ -0,0 +1,207 @@
+import {
```

Comment:
This api fetches all the email data from google client (optionally based on timestamp ), similar to https://github.com/elie222/inbox-zero/blob/main/apps/web/app/api/user/stats/tinybird/load/load-emails.ts.